askama_derive/src/generator.rs

```diff
@@ -665,7 +665,14 @@ impl<'a> Generator<'a> {
                 Node::Include(ws, path) => {
                     size_hint += self.handle_include(ctx, buf, ws, path)?;
                 }
-                Node::Call(ws, scope, name, ref args) => {
+                Node::Call(
```

Comment:
If we're going to do this, I think we should just pass the `&Call` directly to `write_call()`... Not much point in destructing it here. (Same for other node types.)